README.md

```diff
@@ -49,6 +49,8 @@ You can run the test on VSCode
 -  Twitter: [@NonhlanhlaMnde6](https://twitter.com/twitterhandle)
 -  LinkedIn: [LinkedIn](https://linkedin.com/in/nonhlanhla-mndebele-ab7448226)
 
+
+
 ## 🤝 Contributing
 
```

Comment:
- [ ] like the previous reviewer said, The README file is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others. kindly add a brief description of what the project is about.